installer.sh

```diff
@@ -132,6 +134,11 @@ get_option() {
     echo $(grep -E "^${1}.*" $CONF_FILE|sed -e "s|${1}||")
 }
 
+a_lt_b() {
```

Comment:
posix test has `-lt`...

Reply:
hm so the reason is because they didn't know POSIX has a less than operator, which presumably why it's not merged. Compat issues. If that's the case then I'll try to get around to doing a refactoring or a rewrite if that's okay with you.